core/common/src/main/java/alluxio/ClientContext.java

```diff
@@ -123,8 +124,11 @@ public synchronized void loadConf(InetSocketAddress address, boolean loadCluster
     GetConfigurationPResponse response = Configuration.loadConfiguration(address,
         conf, !loadClusterConf, !loadPathConf);
     if (loadClusterConf) {
-      mClusterConf = Configuration.getClusterConf(response, conf, Scope.CLIENT);
+      InstancedConfiguration clusterConf =
+          Configuration.getClusterConf(response, conf, Scope.CLIENT);
+      mClusterConf = clusterConf;
       mClusterConfHash = response.getClusterConfigHash();
+      Configuration.updateNewConf(clusterConf);
```

Comment:
We should not update the static global configuration, as `ClientContext.getClusterConf` is used to get the cluster-global configuration.

Reply:
Can you help to explain why we keep cluster conf and client-side global configuration different?